quickwit/quickwit-config/src/source_config/mod.rs

```diff
@@ -340,7 +340,7 @@ impl TryFrom<KinesisSourceParamsInner> for KinesisSourceParams {
     fn try_from(value: KinesisSourceParamsInner) -> Result<Self, Self::Error> {
         if value.region.is_some() && value.endpoint.is_some() {
             return Err(
-                "Kinesis source parameters `region` and `endpoint` are mutually exclusive.",
+                "kinesis source parameters `region` and `endpoint` are mutually exclusive",
```

Comment:
nit: We can keep uppercase for proper nouns such as `Quickwit`, `Kafka`, `Kinesis`, `Pulsar`, ...